api.go

```diff
@@ -867,6 +912,34 @@ func (lapi *Client) UpdateTaskStatus(id string, phase string, progress float64)
 	return nil
 }
 
+func (lapi *Client) CreateTranscodeTask(assetId string, name string, profile Profile) (*CreateTranscodeTaskResp, error) {
+	var (
+		url    = fmt.Sprintf("%s/api/asset/transcode", lapi.chosenServer)
+		input  = &createTranscodeTaskRequest{AssetId: assetId, Name: name, Profile: profile}
+		output CreateTranscodeTaskResp
+	)
+	err := lapi.doRequest("POST", url, "transcode_task", "", input, &output)
+	if err != nil {
+		return nil, err
+	}
+	glog.V(logs.DEBUG).Infof("Created transcode task id=%s assetId=%s status=%s type=%s", output.Task.ID, output.Asset.ID, output.Task.Status.Phase, output.Task.Type)
+	return &output, nil
+}
+
+func (lapi *Client) CreateExportTask(assetId string) (*CreateExportTaskResp, error) {
+	var (
+		url    = fmt.Sprintf("%s/api/asset/%s/export", lapi.chosenServer, assetId)
+		input  = &createExportTaskRequest{IPFS{}}
+		output CreateExportTaskResp
+	)
+	err := lapi.doRequest("POST", url, "export_task", "", input, &output)
+	if err != nil {
+		return nil, err
+	}
+	glog.V(logs.DEBUG).Infof("Created export task id=%s status=%s type=%s", output.Task.ID, output.Task.Status.Phase, output.Task.Type)
+	return &output, nil
```

Comment:
**nit:** We should probably just return the `output.Task`. No need to expose the internal format of the response that wraps it into an object.